BACKEND/model.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const canteenmenu = new mongoose.Schema({
+    name: String,
+     price: Number,
+     description: String,
+     id:String
+   });
+ module.exports={

```

Comment:
dont export these create model here only and export the model

`module.exports = mongoose.model(Menu", canteenmenu );`